src/main/java/com/arpnetworking/clusteraggregator/Main.java

```diff
@@ -58,14 +59,18 @@
      * @param args command line arguments
      */
     public static void main(final String[] args) {
+        final AtomicReference<DynamicConfiguration> configRef = new AtomicReference<>();
+        final AtomicReference<Configurator<?, ?>> configuratorRef = new AtomicReference<>();
```

Comment:
Let's move these and the first logger statement below the exception handler definitions.